scripts/build_provenance

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+readonly BUILD_SCRIPT=$1
+readonly OUTPUT_PATH=$2
+
+readonly BUILD_COMMAND="./scripts/docker_run ${BUILD_SCRIPT}"
+eval "${BUILD_COMMAND}"
+readonly BINARY_SHA_256_HASH=$(sha256sum "${OUTPUT_PATH}" | cut -d " " -f 1)
+
+readonly COMMIT_HASH=$(git rev-parse --verify HEAD)
+
+mkdir -p "./provenance"
+mkdir -p "./provenance/${BINARY_SHA_256_HASH}"
+
+readonly PROVENANCE=$(cat << EOF >> "./provenance/${BINARY_SHA_256_HASH}/${COMMIT_HASH}.toml"
+repo = "https://github.com/project-oak/oak"
+commit_hash = "$(git rev-parse --verify HEAD)"
+builder_image = "${DOCKER_IMAGE_REPO_DIGEST}"
+command = ["${BUILD_COMMAND}"]
```

Comment:
Shouldn't this be split into the actual args? Or am I missing the point of the array?

Reply:
Frankly there isn't much utility in using an area, and we've decided to move away from it. The only reason this uses an array is legacy, the transparent release parser expects one at the (current) version which this script creates provenance file for.

Reply:
I'll add a comment to make that clearer, agreed the code looks a bit confusing at this point.